### ISSUE TITLE
Idea: Adding interactivity to show model insights on hover

### ISSUE THREAD

**Issue by ajayakhatri** [sewkokot/opsvis#53]
Hi @sewkokot  ! 

Thank you for creating and maintaining this module. It has been a tremendous help in my projects.

I have an idea to enhance the interactivity of the module. Including an option (an optional feature similar to toggling the visibility of axes) that displays insights about the model when users hover over nodes and elements would be beneficial. I have recently developed a script that achieves this and can be seamlessly integrated into opsvis. This script utilizes a JSON file from [`printModel`](https://openseespydoc.readthedocs.io/en/latest/src/printModel.html) function to obtain information on the model's geometry (nodes and elements). In addition, users can add custom texts to any nodes or elements.

What do you think about this feature? I am happy to provide the code and assist with the integration.

![demo](https://github.com/sewkokot/opsvis/assets/69041015/24dd6c87-ef8c-4e08-9501-a6db3cc9c7c7)

Best regards,
Ajaya Khatri

**Comment by ganansuan647** [comment]
Sounds great! Looking forward for this feature

**Comment by GJoe2** [comment]
Any news on this? It looks like a good idea

**Comment by ajayakhatri** [comment]
> Any news on this? It looks like a good idea

Hi, I have added the source code for along with implementation examples in my repo: https://github.com/ajayakhatri/OpsTooltip

Let me know when you think!

**Comment by GJoe2** [comment]
@ajayakhatri  As expected very useful tool, I notice in bigger models a bit of desync between actual cursor position and  tooltip, would you consider using a caching system so it reduces cpu time for retrieving information?

![Image](https://github.com/user-attachments/assets/80bc6c4e-046c-4349-9d1c-8d8a18f60507)

**Comment by ajayakhatri** [comment]
@GJoe2 Good point! If you're up for it, feel free to open an issue in my repo so we can track this there. It'll be easier to follow up and discuss any fixes or improvements in the right context.
